packages/server/src/services/task-work-log.ts

```
@@ -0,0 +1,359 @@
+					id: true,
+					firstName: true,
+					lastName: true,
+				},
+			},
+		},
+	});
+
+	const totalMinutes = logs.reduce((acc, log) => {
+		if (!log.endedAt) {
+			const start = new Date(log.startedAt).getTime();
+			acc += Math.floor((Date.now() - start) / 60_000);
```

Biome (2.3.13)

[error] Assigning a function parameter is confusing.

The parameter is declared here:

Developers usually expect function parameters to be readonly. To align with this expectation, use a local variable instead.

(lint/style/noParameterAssign)